### PR TITLE
Policy files list view tab on org details page

### PR DESCRIPTION
**Pull request by samshinde**
### :nut_and_bolt: Description: What code changed, and why?
Added Infra Policy file tab on the org details page.

### :chains: Related Resources
https://github.com/chef/automate/issues/3406
### :+1: Definition of Done
Added Policy file tab on the org details page where we are displaying the  Policy file lists.

-  When User clicks on the Policy file tab it will show the list of policy files that have Policy Name, Policy Group as it's headings.

### :athletic_shoe: How to Build and Test the Change
```
STEP 1
inside the hab studio

[default:/src:0]# build components/automate-ui-devproxy/
[default:/src:0]# start_automate_ui_background
[default:/src:0]# start_all_services

STEP 2
open new window
go to automate UI path

$ cd components/automate-ui
and run the command 

$npm run serve:hab
```

1. git fetch origin Amol/policy_files_list
2. build components/automate-ui [Above steps for reference.]
3. To add data https://github.com/chef/automate/blob/master/dev-docs/adding-data/adding_test_data.md#adding-data-to-infra-views
4. Go to Infrastructure tab >> Chef Servers its under Chef-Server feature flag.
5. Click on `Server list` >> `Organisation list` >> `Policy files` Tab.
### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable
policylist

### CODE REVIEW

**Comment by msorens** [comment]
Similar to the comment on your other PR, please add details to the instructions in your introduction on how to exercise your code, then I can review. Thanks!

**Comment by samshinde** [comment]
> Similar to the comment on your other PR, please add details to the instructions in your introduction on how to exercise your code, then I can review. Thanks!

@msorens I have updated the description and screenshot of PR.

File: components/automate-ui/src/app/entities/policy-files/policy-file.requests.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Observable } from 'rxjs';
+import { environment as env } from 'environments/environment';
+import { PolicyFilesSuccessPayload } from './policy-file.action';
+
+@Injectable()
+export class PolicyFileRequests {
+
+  constructor(private http: HttpClient) { }
+
+  // tslint:disable-next-line: max-line-length
```
Comment (msorens) [inline components/automate-ui/src/app/entities/policy-files/policy-file.requests.ts:12]: Delete; should not be needed here.

File: components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.ts [hunk 2]
```diff
@@ -103,6 +111,9 @@ export class OrgDetailsComponent implements OnInit, OnDestroy {
         this.telemetryService.track('orgDetailsTab', 'clients');
         break;
       case 5:
+        this.telemetryService.track('orgDetailsTab', 'policyFiles');
+        break;
+      case 6:
         this.telemetryService.track('orgDetailsTab', 'org-edit');
```
Comment (msorens) [inline components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.ts:117]: [Minor] Seems like most of these are camel case but this one ('org-edit') is kebab case. Could we make that 'orgEdit' for consistency perhaps...?

File: components/automate-ui/src/app/modules/infra-proxy/policy-files/policy-files.component.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    this.store.dispatch(new GetPolicyFiles({
+      server_id: this.serverId, org_id: this.orgId
+    }));
+
+    combineLatest([
+      this.store.select(getAllPolicyFilesForOrgStatus),
+      this.store.select(allPolicyFiles)
+    ]).pipe(
+      filter(([getPolicyFilesSt, _allPolicyFilesState]) =>
```
Comment (msorens) [inline components/automate-ui/src/app/modules/infra-proxy/policy-files/policy-files.component.ts:46]: The underscore is only if you are *not* using it.
```suggestion
      filter(([getPolicyFilesSt, allPolicyFilesState]) =>
```

**Comment by samshinde** [comment]
@msorens thanks for the review, I have updated my PR.

File: components/automate-ui/src/app/modules/infra-proxy/policy-files/policy-files.component.ts (hunk 3 above)
Comment (msorens) [reply components/automate-ui/src/app/modules/infra-proxy/policy-files/policy-files.component.ts]: I am still seeing this without the fix...?

**Pull request merged**